.github/workflows/load-test-oha.yml

```
@@ -65,6 +66,59 @@ jobs:
             exit 1
           fi
 
+          # Wait additional 30 seconds for services to stabilize
+          echo "Waiting 30 seconds for services to stabilize..."
+          sleep 30
+
+      - name: Verify Chain ID
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2236:style:25:10: Use -n instead of ! -z

(shellcheck)